### ISSUE TITLE
Keyboard shortcuts in UI

### ISSUE THREAD

**Issue by kwk** [almighty/almighty-ui#16]
To get an idea of what I mean, press `?` outside of a text editor on github:

![screenshot from 2016-08-25 16-32-57](https://cloud.githubusercontent.com/assets/193408/17973191/bde581c4-6ae1-11e6-8469-7920dd5f2e00.png)

**Comment by bartoszmajsak** [comment]
Or gmail or twitter or confluence.

Big +1 @kwk

**Comment by joshuawilson** [comment]
now that will take some thought or a library ;)

**Comment by bartoszmajsak** [comment]
I don't think it requires that much of work to be honest, let it be with or w/o the library. If this will show up as a requirement at some point I would be very happy to take this task :)

**Comment by aslakknutsen** [comment]
The work is in defining a consistent set of shortcuts.. ☺
